test/blackbox-tests/test-cases/watching/multiple-errors.t

```diff
@@ -0,0 +1,39 @@
+We test the behavior of watch mode when we have multiple errors
+
+  $ . ./helpers.sh
+
+  $ export PATH=.:$PATH
```

Comment:
Why do you modify the PATH?

Reply:
That was copied from a test, I'll remove it.